buku

```diff
@@ -95,6 +95,8 @@ COLORMAP = {k: '\x1b[%sm' % v for k, v in {
     'x': '0', 'X': '1', 'y': '7', 'Y': '7;1', 'z': '2',
 }.items()}
 
+[FLAG_NONE, FLAG_IMMUTABLE] = [0x00, 0x01]
```

Comment:
They're bitmasks which are used as magic numbers in the current master (both in API and in DB). I simply provided explicit names for them here. And yes, they're directly related – they're values assigned to the `flags` field in DB (I imagine it can be migrated to a boolean field should you prefer it, though I'm not sure how viable this is, but my [first attempt at this](https://github.com/LeXofLeviafan/buku/commit/8d2eb1392283d779c6ea972acbccb3b77d0b3ba9) had indeed converted it to bool across the API).

Reply:
OK. This is fine. Just add a  comment above that says - various DB flags.